{"name": "Принятие заказа без ID курьера", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Недостаточно данных для входа' in 'Недостаточно данных для поиска'", "trace": "self = <tests.test_accept_order.TestAcceptOrder object at 0x00000284B9516FD0>, setup_courier_and_order = {'courier_id': 615722, 'login': 'yzcmhhilcv', 'order_id': 546569, 'password': 'bjvdfdqvtq', ...}\n\n    @allure.title(\"Принятие заказа без ID курьера\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_accept_order_without_courier_id(self, setup_courier_and_order):\n        with allure.step(\"Попытаться принять заказ без ID курьера\"):\n            response = OrderApi.accept_order(\n                setup_courier_and_order[\"order_id\"],\n                \"\"\n            )\n    \n        with allure.step(\"Проверить ошибку недостатка данных\"):\n            assert response.status_code == ExpectedResponses.NOT_ENOUGH_DATA[\"code\"]\n>           assert ExpectedResponses.NOT_ENOUGH_DATA[\"message\"] in response.json()[\"message\"]\nE           AssertionError: assert 'Недостаточно данных для входа' in 'Недостаточно данных для поиска'\n\ntests\\test_accept_order.py:36: AssertionError"}, "steps": [{"name": "Попытаться принять заказ без ID курьера", "status": "passed", "steps": [{"name": "Принять заказ 546569 курьером ''", "status": "passed", "parameters": [{"name": "order_id", "value": "546569"}, {"name": "courier_id", "value": "''"}], "start": 1758527528785, "stop": 1758527529216}], "start": 1758527528785, "stop": 1758527529216}, {"name": "Проверить ошибку недостатка данных", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Недостаточно данных для входа' in 'Недостаточно данных для поиска'\n", "trace": "  File \"C:\\Users\\Пользователь\\sprint_7\\tests\\test_accept_order.py\", line 36, in test_accept_order_without_courier_id\n    assert ExpectedResponses.NOT_ENOUGH_DATA[\"message\"] in response.json()[\"message\"]\n"}, "start": 1758527529216, "stop": 1758527529219}], "start": 1758527528785, "stop": 1758527529221, "uuid": "bf7647eb-8a6e-4e4b-9255-96d251168efd", "historyId": "e671566bdda014251dc5f006893cbb51", "testCaseId": "e671566bdda014251dc5f006893cbb51", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_without_courier_id", "labels": [{"name": "epic", "value": "Order Management"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Accept Order API"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "9396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}